### ISSUE TITLE
NTR: N-acetyltyrosine high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8576]
New term request N-acetyltyrosine high in urine (segal_030509153832):

**Comment by pnrobinson** [comment]
@MickeySegal
is this a clinical assay? What is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1515/ "Increased excretion of p-hydroxyphenylpyruvate, p-hydroxyphenyllactate, and p-hydroxyphenylacetate and presence of small quantities of N-acetyltyrosine and 4-tyramine on urine organic acid analysis"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000479
- label: Elevated urine N-acetyltyrosine level
- parents: [Increased aromatic amino acid level in urine (HP:0033090)](https://hpo.jax.org/app/browse/term/HP:0033090)
- pmid: PMID:20301688
- def: The amount of N-acetyltyrosine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8576
- synonyms:
